pkg/client/client.go

```diff
@@ -159,6 +168,8 @@ type ImmuClient interface {
 	DescribeTable(ctx context.Context, tableName string) (*schema.SQLQueryResult, error)
 
 	VerifyRow(ctx context.Context, row *schema.Row, table string, pkVals []*schema.SQLValue) error
+
+	BeginTx(ctx context.Context, opts *TxOptions) (Tx, error)
```

Comment:
I still think it's rare this API, having BeginTx at this level and Commit or Close in the Tx.. @byo ?

Reply:
Done